cv32e20/sim/ExternalRepos.mk

```diff
@@ -12,7 +12,7 @@ export SHELL = /bin/bash
 
 CV_CORE_REPO   ?= https://github.com/openhwgroup/cve2
 CV_CORE_BRANCH ?= main
-CV_CORE_HASH   ?= 00f7eb1
+CV_CORE_HASH   ?= 9e0615b
```

Comment:
This version of the core has an additional interrupt input.  The testbench does not currently drive this.

Reply:
I will implement your suggested change to the interrupts to drive the extra interrupt input.